metadata_service/proxy/atlas_proxy.py

```diff
@@ -736,6 +739,31 @@ def _get_readers(self, qualified_name: str, top: Optional[int] = 15) -> List[Rea
 
         return results
 
+    def _get_programmatic_descriptions(self, parameters: dict) -> List[ProgrammaticDescription]:
+        _programmatic_descriptions = {}
+        programmatic_descriptions: List[ProgrammaticDescription] = []
+
+        for source, text in parameters.items():
+            use_parameter = True
+
+            for f in app.config['PROGRAMMATIC_DESCRIPTIONS_EXCLUDE_FILTERS']:
```

Comment:
nit: `for filter in app.config['PROGRAMMATIC_DESCRIPTIONS_EXCLUDE_FILTERS']:`

Reply:
`filter` shadows built-in name, I've changed to `regex_filter`